### PR TITLE
Feature/headers

### PR DESCRIPTION
**Pull request by mooyoul**
## New Features

1. Normalizes request headers (and works lazily & cached for futher use). Developers can access request headers via `RoutingContext.headers`.
2. Add ability to insert response headers to `RoutingContext.json`
3. Add CORS support to `SwaggerRouter`.

### CODE REVIEW

**Comment by mooyoul** [comment]
+Bonus: Make readme prettier

File: src/routing-context.ts [hunk 1]
```diff
@@ -77,6 +79,12 @@ export class RoutingContext {
     };
   }
 
+  get headers(): LambdaProxy.EventHeaders {
+    // normalize works lazily and should be cached for further use
+    return this.normalizedHeaders
```
Comment (breath103) [inline src/routing-context.ts]: this.normalizedHeaders is not updated

File: src/routing-context.ts [hunk 2]
```diff
@@ -14,12 +14,14 @@ const DefaultJoiValidateOptions = {
 // ---- RoutingContext
 export class RoutingContext {
   private validatedParams: { [key:string]: any };
+  private normalizedHeaders: { [key: string]: any };
```
Comment (breath103) [inline src/routing-context.ts]: Headers's type will be always { [key: string]: string }

**Comment by breath103** [comment]
Can you update package version (minor update) @mooyoul

**Comment by breath103** [comment]
:+1:

**Pull request merged**